### PR TITLE
User13 homepage

### PR DESCRIPTION
**Pull request by ValerieVT**

### CODE REVIEW

File: src/components/Footer.jsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+const Footer = () => {
+  return (
+    <footer>
+      <ul>
+        <li>
+          <a href="/en-savoir-plus">En savoir plus</a>
+        </li>
+        <li>
+          <a href="/contact">Nous contacter</a>
+        </li>
```
Comment (EmilieM71) [inline src/components/Footer.jsx]: Replace links with "link to"

File: src/components/FirstFilters.jsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  return (
+    <article>
+      <div className={styles.bigButton}>
+        <Link to="/jeveuxtrouver" title="">
+          Je veux trouver THE film&nbsp;!
+        </Link>
+      </div>
+      <div className={styles.bigButton}>
+        <Link to="/filmChoice" title="Je partage le film que je veux regarder">
+          J’ai trouvé&nbsp;: je partage&nbsp;!
```
Comment (EmilieM71) [inline src/components/FirstFilters.jsx]: Replace "trouvé" by "l'idée"

**Review by EmilieM71 — CHANGES_REQUESTED** [review]

File: src/components/Footer.jsx (hunk 1 above)
Comment (ValerieVT) [reply src/components/Footer.jsx]: Ok, done for the next push (with title attributes inside, that I forgot before).

File: src/components/FirstFilters.jsx (hunk 2 above)
Comment (ValerieVT) [reply src/components/FirstFilters.jsx]: Ok, I replaced it by "J'ai déjà l'idée : je partage !". Is this okay with you ?

Comment (ValerieVT) [reply src/components/FirstFilters.jsx]: resolved on this commit : https://github.com/WildCodeSchool/remotefr-js-0920-p2-themoviedb/pull/1/commits/a3291347310956dcf5af255bd892e028cc04dfad

File: src/components/Footer.jsx (hunk 1 above)
Comment (ValerieVT) [reply src/components/Footer.jsx]: You can see that modification on this commit: https://github.com/WildCodeSchool/remotefr-js-0920-p2-themoviedb/pull/1/commits/66840d2c1332d317c1b41cfd0c77f239022663a2

**Review by EmilieM71 — APPROVED** [review]

**Pull request merged**